EventFiltering/PWGCF/CFFilter.cxx

```diff
@@ -78,25 +80,45 @@ struct CFFilter {
 
   Produces<aod::CFFilters> tags;
 
-  //Obtain particle and antiparticle candidates of protons and lambda hyperons for current femto collision
+  Configurable<std::vector<float>> confQ3TriggerLimit{"Q3TriggerLimitC", std::vector<float>{0.6f, 0.6f, 0.6f, 0.6f}, "Q3 limit for selection"};
+  Configurable<int> Q3Trigger{"Q3Trigger", 0, "Choice which trigger to run"};
+  Configurable<float> ldeltaPhiMax{"ldeltaPhiMax", 0.017, "Max limit of delta phi"};
+  Configurable<float> ldeltaEtaMax{"ldeltaEtaMax", 0.017, "Max limit of delta eta"};
+  Configurable<float> lmagfield{"lmagfield", 0.5, "Magnetic field value"};
```

Comment:
You don't need a configurable for the magnetic field any longer
Have a look at https://github.com/AliceO2Group/O2Physics/blob/master/PWGCF/Tasks/correlations.cxx#L162

Reply:
Thanks a lot for pointing to this example, Victor! Vale is implementing this now, but I want to merge the trigger code now as it is so that I can test it on hyperloop over the weekend. Next week we will include the magnetic field information appropriately.